test/controllers/agent/ManageAgentControllerSpec.scala

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.agent
+
+import actions.AuthenticatedAction
+import config.ApplicationConfig
+import controllers.VoaPropertyLinkingSpec
+import controllers.agent.ManageAgentController
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.when
+import org.scalatest.mockito.MockitoSugar
+import play.api.i18n.MessagesApi
+import play.api.mvc.MessagesControllerComponents
+import play.api.test.{FakeRequest, Helpers}
+import play.api.test.Helpers._
+import play.twirl.api.Html
+import services.AgentRelationshipService
+import tests.AllMocks
+import uk.gov.hmrc.propertylinking.errorhandler.CustomErrorHandler
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ManageAgentControllerSpec extends VoaPropertyLinkingSpec with MockitoSugar with AllMocks {
```

Comment:
This doesn't look like most of our tests. It doesn't have the same layout/structure. No shared setups. No `when`s

Reply:
It does. please have a look at other tests in voa-property-linking-frontend

Reply:
I was referring to new standards, not old ones :)